lib/ClangImporter/swift-interop-support.h

```diff
@@ -28,7 +28,7 @@
 
 #define SWIFT_NAME(_name) __attribute__((swift_name(#_name)))
 
-#define CONFORMS_TO(_name) \
-  __attribute__((swift_attr(_CXX_INTEROP_STRINGIFY(conforms_to:_name))))
+#define CONFORMS_TO(_moduleName,_name) \
+  __attribute__((swift_attr(_CXX_INTEROP_STRINGIFY(conforms_to:_moduleName._name))))
```

Comment:
It seems much more natural to type `CONFORMS_TO(Swift.Equatable)` than `CONFORMS_TO(Swift, Equatable)`. Especially if we don't have any labels, I think users will not know what to put for the first argument. If there's only one argument, they will run into your nice error message telling them they need to specify the module name. If there are two arguments, they won't have anything actionable to fix, it will just say "macro expected two arguments."

Reply:
good idea.